src/Sylius/Behat/Context/Setup/TaxContext.php

```diff
@@ -51,14 +52,14 @@
      * @param FactoryInterface $taxRateFactory
      * @param FactoryInterface $taxCategoryFactory
      * @param RepositoryInterface $taxRateRepository
```

Comment:
Tax rate doesn't have custom repository. But good question, I found some other places where it wasn't updated.